.gitignore

```diff
@@ -2,4 +2,5 @@
 .idea/
 source/.DS_Store
 node_modules/
-build/
\ No newline at end of file
+build/
+Makefile
```

Comment:
If we are going to ignore it do we want to check it into git?

Reply:
Ignore.